projects/canopy/src/lib/side-nav/side-nav-bar-item/side-nav-bar-item.component.scss

```diff
@@ -54,8 +54,9 @@
 
 .lg-side-nav-bar-item--selected {
   .lg-side-nav-bar-item__inner {
-    color: var(--color-super-blue);
-    border-bottom: var(--side-nav-bar-item-border-width-md) solid var(--color-super-blue);
+    color: var(--side-nav-bar-active);
+    border-bottom: var(--side-nav-bar-item-border-width-md) solid
+      var(--side-nav-bar-active);
```

Comment:
In this case I think it's fine to use the same variable as above `--side-nav-bar-active-color`